src/css/var/getStyles.js

```diff
@@ -1,7 +1,7 @@
 define( function() {
 	return function( elem ) {
 
-		// Support: IE<=11 only, Firefox<=30+ (#15098, #14150)
+		// Support: IE <=11 only, Firefox <=30+ (#15098, #14150)
```

Comment:
`Firefox <=30+` isn't valid. If the issue still exists, it should be `Firefox <=30 - 45`.